Update ChartJs.php

kanyuga opened the pull request:
Set the `assetsPath` relative to the file itself. This fixes a bug which caused `chartjs/assets` folders to be added to the `myApp/assets` folder recursively. Also eliminates the need for a `chartjs` alias.

MeiSign commented:
Can you please give me instructions how I can reconstruct this bug? I think an alias is a cleaner way than using a relative path.

kanyuga commented:
I placed the yii-chartjs folder in extensions and renamed it to chartjs. I then edited the `config/main.php` and added `aliases`, `preload` and `components` snippets according to the instructions. The page exceeded the 30 second limit on my setup. On investigating I found that the `chartjs/assets` folder had been recursively copied into `MyApp/assets/xxxxxx/`. I'm on Ubuntu 12.04 64-bit with PHP>5.3 if that helps.

Using the updated code, I left the `preload` and `components` snippets and deleted the contents of `MyApp/assets`. The recursive copying of the `chartjs/assets` folder didn't occur then.
